src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -251,4 +408,23 @@ private function getNewOrder(array $uses, Tokens $tokens)
 
         return $usesOrder;
     }
+
+    /**
+     * @param $indexes
+     *
+     * @return array
+     */
+    private function sortByAlgorithm($indexes)
+    {
+        switch ($this->sortAlgorithm) {
+            case self::SORT_LENGTH:
+                uasort($indexes, array($this, 'sortByLength'));
+                break;
+            default:
```

Comment:
But we already checking if correct sort type is passed in configuration: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2280/files/da3106b7fbb3249ad15e8698905805acf933efd7#diff-80d6131ebed1ab07edef75d2d7ad6c82R120

and by default, we are sorting alphabetically